src/tuvx/interface_util.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2023-2024 National Center for Atmospheric Research
+// SPDX-License-Identifier: Apache-2.0
+//
+// This file contains hepler functions for the internal TUV-x interface.
+#pragma once
+
+#include <cstring>
+#include <musica/util.hpp>
+
+namespace musica
+{
+
+#ifdef __cplusplus
+  extern "C"
+  {
+#endif
+
+  /// @brief Create a string from a C-style string
+  /// @param str C-style string
+  /// @return String struct
+  String InternalCreateString(const char *str);
```

Comment:
Same question as above. Also, does it make sense to locate this header file in `include/musica/tuvx`?

Reply:
removed!